packages/axios-extension/src/abap/adt-catalog/services/transportcheck-service.ts

```diff
@@ -65,6 +65,91 @@ export class TransportChecksService extends AdtService {
         return this.getTransportRequestList(response.data);
     }
 
+    /**
+     * TransportChecksService API function to fetch package and transport of deployed application.
+     *
+     * @param namespace The namespace for the deployed application
+     * @returns Object that contains the package and the tansport used for the application deployment
+     */
+    public async transportCheck(namespace: string): Promise<TransportCheck> {
```

Comment:
The function name should always start with a verb. More importantly though, please compare your new function with `getTransportRequests` and let's discuss how to consolidate these. On first glance, they do the same, just that one checks for a BSP and the other for a LREP entry

Reply:
We won't need this method because it is used in our Deployment Generator and after discussions its prompting won't be migrated in open-ux-tools.